src/core/RadioButton/index.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import useStyles from './styles';
+import Radio, { RadioProps } from '@material-ui/core/Radio';
```

Comment:
Also we need to put RadioProps into base.ts file and take it from there
base.ts will look like this
```
import { RadioProps } from '@material-ui/core';

export type RadioBaseProps = RadioProps;
```